### PR TITLE
Cu 6mudw6/performance

### PR DESCRIPTION
**Pull request by hobadams**
Small performance improvements + docusaurus version update

### CODE REVIEW

**Comment by cane-lagerwaard** [comment]
Task linked: [CU-6mudw6 Improve Falcon Server docs](https://app.clickup.com/t/6mudw6)

File: docusaurus.config.js [hunk 1]
```diff
@@ -159,13 +159,7 @@ module.exports = {
...
         containerId: 'GTM-N4NQJGX',
       },
-    ],
-    [
-      path.resolve(__dirname, './src/plugins/deity-intercom/src/index.js'),
-      {
-        appId: 'z91ewqiv',
-      },
-    ],
+    ]
```
Comment (Borales) [inline docusaurus.config.js:162]: @hobadams why did you move the config to the component?

Comment (hobadams) [reply docusaurus.config.js:165]: @Borales - good point. I've moved it back into the config (but not as a plugin)

File: src/components/Intercom/Chat.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+export const Chat = () => {
+  const context = useDocusaurusContext();
+  const {
+    siteConfig: {
+      themeConfig: {
+        intercom: { appId },
+      },
+    },
+  } = context;
```
Comment (Borales) [inline src/components/Intercom/Chat.js:46]:
```suggestion
  const appId = context.siteConfig?.themeConfig?.intercom?.appId;
```

Does docusaurus support this kind of syntax? If we gonna remove the whole `intercom` from the config - the build/start will fail completely. Using optional chaining will protect us from it.

**Comment by hobadams** [comment]
@Borales - optional chaining is supported..nice one :-)

**Comment by hobadams** [comment]
@Borales - are you ok me merging this?

**Comment by Borales** [comment]
@hobadams yes!
